src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1110,11 +1185,11 @@ public function guessMimeType(string $path): ?string
         'audio/m3u' => ['m3u', 'm3u8', 'vlc'],
         'audio/m4a' => ['m4a', 'f4a'],
         'audio/midi' => ['mid', 'midi', 'kar', 'rmi'],
-        'audio/mobile-xmf' => ['xmf'],
+        'audio/mobile-xmf' => ['mxmf', 'xmf'],
         'audio/mp2' => ['mp2'],
         'audio/mp3' => ['mp3', 'mpga'],
         'audio/mp4' => ['m4a', 'mp4a', 'f4a'],
-        'audio/mpeg' => ['mp3', 'mpga', 'mp2', 'mp2a', 'm2a', 'm3a'],
+        'audio/mpeg' => ['mpga', 'mp2', 'mp2a', 'mp3', 'm2a', 'm3a'],
```

Comment:
related to https://github.com/symfony/symfony/pull/37389

Reply:
Yes, it's the most common format. You can also add `mp1` like defined [here (page3)](https://tools.ietf.org/html/rfc3003).

Reply:
@YaFou could you please report this to https://github.com/jshttp/mime-db? Maybe send them a PR also?

Reply:
I will do that this evening!

Reply:
@nicolas-grekas: see jshttp/mime-db#201.

Reply:
@YaFou can you revert this change, maybe adding a comment like `// mp3 should be first`
same in other places where I commented about ordering?
that'd allow unlocking this PR

Reply:
I meant @desarrolla2 sorry :

Reply:
@nicolas-grekas do you want that:

a) I put the extensions in the order that you comment at MimeTypes.php
b) I update the script update_mime_types.php to force that order after download sources
c) I add comments with your suggestions at MimeTypes.php

Reply:
I think b) would be good. Then the comments would be found in update_mime_types.php only.